src/main/java/net/sourceforge/tess4j/util/LoadLibs.java

```diff
@@ -15,24 +15,29 @@
  */
```

Comment:
@nguyenq now here comes in the changes that are actually in the scope and concern of this merge request.